pkg/hubble/parser/seven/parser.go

```diff
@@ -157,6 +173,18 @@ func (p *Parser) computeResponseTime(r *accesslog.LogRecord, timestamp time.Time
 	return 0
 }
 
+func (p *Parser) updateEndpointWorkloads(ip net.IP, endpoint *pb.Endpoint) {
+	if ep, ok := p.endpointGetter.GetEndpointInfo(ip); ok {
+		if pod := ep.GetPod(); pod != nil {
+			olen := len(pod.GetOwnerReferences())
+			endpoint.Workloads = make([]*pb.Workload, olen)
+			for index, owner := range pod.GetOwnerReferences() {
```

Comment:
We're calling `pod.GetOwnerReferences()` twice here - I see it's same as for L3/4, but it looks unnecessary